src/schema/v2/partner/partner.ts

```diff
@@ -440,7 +449,11 @@ export const PartnerType = new GraphQLObjectType<any, ResolverContext>({
             size,
             sort: args.sort,
             total_count: true,
+            visibility_levels: args.visibilityLevels || [
```

Comment:
🐛 here, the fallback case is taking over even if I'm passing args into the function

This is not ruby 😂 , fixing